test/JsBuiltIns/ChakraLib.js

```diff
@@ -0,0 +1,17 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+WScript.LoadScriptFile("..\\UnitTestFramework\\UnitTestFramework.js");
+
+var tests = [
+    {
+        name: "Access to __chakraLibrary work",
+        body: function () {
+            assert.isNotUndefined(__chakraLibrary, "__chakraLibrary must NOT be undefined as we're using the -LdChakraLib flag.");

```

Comment:
New test LGTM, thanks.

---
In reply to: [116263296](https://github.com/Microsoft/ChakraCore/pull/2898#discussion_r116263296) [](ancestors = 116263296,115902571)